ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__azure-4.17-nightly-x86.yaml

```diff
@@ -34,12 +34,13 @@ tests:
       CUSTOM_AZURE_REGION: centralus
       PROFILE_TYPE: reporting
       SET_ENV_BY_PLATFORM: custom
-      WORKER_REPLICA_COUNT: "120"
```

Comment:
Please update the job name from `control-plane-ipsec-120nodes` --> `control-plane-ipsec-60nodes`

Reply:
@krishvoor thanks, I'll change it back when the tests are done, using 60 to save some time and resource at early phase.